Big Gemini

ctiberious opened the pull request:
Update and setup various parts for the Big Gemini spacecraft.  Changes include:
1) Standard Big Gemini crew module crew capacity increased to 7.  This allows a 9-man Big Gemini setup (including two man cabin).
2) Rescue (White) Big Gemini crew module crew capacity increased to 10.  This allows 12-man Big Gemini setup (including two man cabin).
3) Both Big Gemini crew modules have the same tank capacity.  The white module uses all it's capacity for life support.  Since the black module allows for fewer crew, it has some extra tank space available which represents this module having some additional storage space.
4) Big Gemini Retro Module has all life support and equipment removed.  It is not setup just as the retro pack, just like the standard retro module for the normal Gemini.  I did include the ConnectedLivingSpace module, though, so that crew can pass through the retro module to make use of the rear docking port.
5) Big Gemini Service Module graphics have been realigned and oriented.  You no longer have to flip the part to set it up for use.
6) BGSM no longer includes science or crew components.
7) BGSM Storage capacity (i.e., Volume) has been increased significantly to account for the 1310 cu ft (37095.07L) of cargo space the module appears to have been capable of(see illustration at http://www.astronautix.com/graphics/b/bigcutaw.jpg)
8) BGSM includes ConnectedLivingSpace module so crew can make use or rear docking port
9) It looks like the BGSM as originally added to RO may have been intended to include built in engines but they didn't work.  I've removed those modules and added in the RCS module to allow for forward orbital translation to occur.
10) Big Gemini Docking Ring has been resized and adjusted similar to the FASAApollo_DockingDevice.  It also includes a ConnectedLivingSpace module.
11) Big Gemini Decoupler has been correctly sized for the 6.6m base.
12) I've cleaned up and balanced out the various Big Gemini life support and fuel cell modules so they properly maintain mass.

ctiberious commented:
I've adjusted the built in life support supplies on the Gemini to account for a max 14 day flight.  I also rebalanced the CO2 Scrubber so it processes CO2 at the same rate as Apollo while correclty maintaining mass.
I removed the tank and LS changes that the Rescue (White) pod had since the pod itself was essentially the same as the normal pod.
I altered the fuel cell and LOX-O2 generator on the utility pack so that they maintain proper mass.  Also increased the available volume to account for these alterations.

ctiberious commented:
While verifying that mass was being properly maintained on the Gemini parts, I noticed an issue with the Apollo CO2 Scrubber.  I've corrected that and slightly increased the available waste storage on the Apollo CM to account for the discrepancy.

ctiberious commented:
Think I only used "x-man" in two places (the two Big G crew modules) so those are both fixed now.

Yes, the FASAGeminiBigGDock is not setup to only be passable.  You can't put crew into it.  The only reason it has any Oxygen supply at all is for the LOX-O2 converter to function correctly.

Only thing that may still need to be adjusted is the strength of the RCS thrusters on the FASABigGeminiRetroModule.  Those come directly from the FASAGeminiUtilitySasRcs which is designed to handle maneuvering for the 3.5t Gemini.  They may be under powered for the 15-50t Big G but I wanted to run a few more tests before tweeking those values.

stratochief66 commented:
Sounds fair. So we shouldn't merge this yet? Please leave a message here
when you're fine with it being merged

On Thu, Mar 10, 2016 at 6:45 PM, ChrisL <notifications@github.com> wrote:

> Think I only used "x-man" in two places (the two Big G crew modules) so
> those are both fixed now.
>
> Yes, the FASAGeminiBigGDock is not setup to only be passable. You can't
> put crew into it. The only reason it has any Oxygen supply at all is for
> the LOX-O2 converter to function correctly.
>
> Only thing that may still need to be adjusted is the strength of the RCS
> thrusters on the FASABigGeminiRetroModule. Those come directly from the
> FASAGeminiUtilitySasRcs which is designed to handle maneuvering for the
> 3.5t Gemini. They may be under powered for the 15-50t Big G but I wanted to
> run a few more tests before tweeking those values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KSP-RO/RealismOverhaul/pull/1050#issuecomment-195103939>
> .
>

ctiberious commented:
I'm changing the texture used for the Big G Retro Module.  The original texture didn't handled the 3.9m to 4.6m taper as well as the texture I'm using now.  I also read (http://www.astronautix.com/craft/bigemini.htm) that the retro module only included the retro engines, plus water and oxygen.  It didn't include any attitude control thrusters like the comparable module on the standard Gemini setup.  I've therefore removed the RCS module from this part and I've added a 2nd "fuel tank" setup specifically for life support to store the bulk of the flights water supply and backup O2.  This module includes textures for four "Gemini Mini SRB" to act as the nozels for the retro engines.  About the only real issue with this texture (that I can see) is that it is no longer "hollow".

I'm changing the texture used on the Big G Service Module so that we now only use the FASAGeminiBigGDock graphic.  I also added four Apollo RCS thruster textures since the same page I cited above says the service module includes the attitude control system.  The illustrations I've found seem to agree.  I've also removed the bulk of the water from this module (moved to the retro module) and reduced the storage volume accordingly.  The ModuleRCS thrust on this part has been doubled as compared to the RCS on a standard Gemini.  The tests I've run so far seem to work well with this thrust setting.

Lastly, I've altered the size of the Big Gemini Docking Ring to more closely match the dimensions I'm seeing on various illustrations (see above website).

At this point, I think this PR should be good to go.
